src/components/Reports/ProjectMemberTable/ProjectMemberTable.jsx

```diff
@@ -86,23 +86,25 @@ export const ProjectMemberTable = ({ projectMembers, skip, take, handleMemberCou
     <div className="project-member-table test">
       <h5 className="project-member-table-title">Members</h5>
       <div className="project-member-count-head">
-        <div className="filter-members-mobile"
-          onChange={e => {
-            setMemberFilter(e.target.value);
-          }}
-        >
-          <input type="radio" name="memberFilter" value="active" id="active" defaultChecked />
-          <label htmlFor="active" id="project-active-member-count" className="project-member-count">
-            ACTIVE: {foundUsers.length}
-          </label>
-          <input type="radio" name="memberFilter" value="all-time" id="all-time" />
-          <label htmlFor="all-time" id="project-all-member-count" className="project-member-count">
-            ALL-TIME: {members.length}
-          </label>
-          {memberFilter == 'all-time'
-            ? handleMemberCount(allMemberList.length)
-            : handleMemberCount(activeMemberList.length)}
-        </div>
+      <div className="filter-members-mobile"
+        onChange={e => {
+          setMemberFilter(e.target.value);
+          if (e.target.value === 'all-time') {
+            handleMemberCount(allMemberList.length);
+          } else {
+            handleMemberCount(activeMemberList.length);
+          }
+        }}
+      >
+        <input type="radio" name="memberFilter" value="active" id="active" defaultChecked />
+        <label htmlFor="active" id="project-active-member-count" className="project-member-count">
+          ACTIVE: {foundUsers.length}
+        </label>
+        <input type="radio" name="memberFilter" value="all-time" id="all-time" />
+        <label htmlFor="all-time" id="project-all-member-count" className="project-member-count">
+          ALL-TIME: {members.length}
+        </label>
+      </div>
```

Comment:
Could you clarify the intent behind these changes? I tested it with [Orientation and Initial Setup](http://localhost:3000/projectreport/6157b9a66dfe943665c9f83b) project.

The pagination doesn't show up automatically after loading member list. I feel the bug is introduced by this change.


Your code:
<img width="758" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/59299011/d2cfce87-a291-4faa-abe5-90e55b9e84a1">

Dev branch:
<img width="770" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/59299011/cf1b57e4-bfa9-45a2-b0b7-7100e9ac0c10">

Reply:
I also see the bug you are talking about regarding the pagination. Will fix it.